src/build.py

```diff
@@ -691,17 +691,27 @@ def LLVM():
              '-DLLVM_ENABLE_ASSERTIONS=ON',
              '-DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=WebAssembly',
              '-DLLVM_TARGETS_TO_BUILD=X86']
-  ccache = Which('ccache')
-  if ccache:
-      command.extend(['-DCMAKE_%s_COMPILER_LAUNCHER=%s' %
-                      (c, ccache) for c in ['C', 'CXX']])
+
+  compiler_launcher = None
+  jobs = []
+  if 'GOMA_DIR' in os.environ:
+    compiler_launcher = os.path.join(os.environ['GOMA_DIR'], 'gomacc')
```

Comment:
Hm, maybe not? https://wasm-stat.us/builders/linux/builds/11019/steps/LLVM/logs/stdio